src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -6,21 +6,24 @@ import * as ssn from '../../common/schemaform/definitions/ssn';
 
 import { relationshipLabels, genderLabels } from '../utils/helpers';
 
-const defaults = {
-  fields: [
-    'relativeFullName',
-    'relativeSocialSecurityNumber',
-    'view:noSSN',
-    'relativeDateOfBirth',
-    'gender',
-    'relationship'
-  ],
-  required: [
-    'relativeFullName',
-    'relativeDateOfBirth',
-    'relationship'
-  ],
-  labels: {}
+const defaults = (prefix) => {
```

Comment:
Still makes me happier than it rightly should. Just...the functionality is so...DRY!